app/g_lfind.mlg

```diff
@@ -2,18 +2,16 @@ DECLARE PLUGIN "lfind_plugin"
 
 {
 open Ltac_plugin
+open Stdarg
+open Tacarg
 }
 
 TACTIC EXTEND lfind_tac
-| [ "lfind" ] -> { Lfind_main.lfind_tac false "coqsynth"}
-| [ "lfind_coqsynth" ] -> { Lfind_main.lfind_tac false "coqsynth"}
-| [ "lfind_myth" ] -> { Lfind_main.lfind_tac false "myth"}
+| [ "lfind" ] -> { Lfind_main.lfind_tac false}
```

Comment:
You could also make the debugging flag as a Opt. So we would just have lfind as the tactic here!

Reply:
I had tried to do this but it kept not registering the changes from including Set Lfind Debug in the coq file (I had implemented it similar to how the other boolean options were implemented). Need to look into that more.